docs/iris/src/further_topics/lenient_metadata.rst

```diff
@@ -0,0 +1,477 @@
+:ref:`lenient difference` is enabled throught the ``lenient`` keyword argument.
+
+The difference in behaviour between **lenient** and
+:ref:`strict combination <strict combine table>` is centered around the lenient
+handling of combining **something** with **nothing** (``None``) to return
+**something**. Whereas strict
+combination will only return a result from combining identical objects.
+
+Again, this is best demonstrated through a simple example of attempting to combine
+partially overlapping ``attributes`` member dictionaries. For example, given the
+following ``attributes`` dictionary of our favoured ``latitude`` coordinate,
+
+.. doctest:: lenient-behaviour
+
+    >>> latitude.attributes  # doctest: +SKIP
+    {'grinning face': '😀', 'neutral face': '😐'}
+
+We create another :class:`~iris.common.metadata.DimCoordMetadata` with overlapping
+keys and values, namely,
+
+- the ``grinning face`` key is **missing**,
+- the ``neutral face`` key has the **same value**, and
+- the ``upside-down face`` key is **new**
+
+.. doctest:: lenient-behaviour
+
+    >>> attributes = {"neutral face": "😐", "upside-down face": "🙃"}
+    >>> metadata = latitude.metadata._replace(attributes=attributes)
+    >>> metadata  # doctest: +SKIP
+    DimCoordMetadata(standard_name='latitude', long_name=None, var_name='latitude', units=Unit('degrees'), attributes={'neutral face': '😐', 'upside-down face': '🙃'}, coord_system=GeogCS(6371229.0), climatological=False, circular=False)
+
+Comparing the strict and lenient behaviour of ``combine`` side-by-side
+highlights the difference in behaviour, and the advantages of lenient combination
+for more inclusive, richer metadata,
+
+.. doctest:: lenient-behaviour
+
+    >>> metadata.combine(latitude.metadata).attributes
+    {'neutral face': '😐'}
+    >>> metadata.combine(latitude.metadata, lenient=True).attributes  # doctest: +SKIP
+    {'neutral face': '😐', 'upside-down face': '🙃', 'grinning face': '😀'}
+
+
+.. _lenient members:
+
+Lenient members
+---------------
+
+:ref:`lenient behaviour` is not applied regardlessly across all metadata members
+participating in a lenient ``equal``, ``difference`` or ``combine`` operation.
+Rather, a more pragmatic application is employed based on the `CF Conventions`_
+meaning of the member, and whether being lenient would result in erroneous
+behaviour or interpretation.
+
+.. _lenient members table:
+.. table:: - Lenient member participation
+   :widths: auto
+   :align: center
+
+   ============================================================================================= ================== ============
+   Metadata Class                                                                                Member             Behaviour
+   ============================================================================================= ================== ============
+   †                                                                                             ``standard_name``  ``lenient``‡
```

Comment:
Seems a bit strange to just have the dagger here. Why not have something like
`All metadata objects†`

And then keep the key as it was before